### PR TITLE
/invis

### PR DESCRIPTION
**Pull request by LegendIsAwesomes**
Resolves https://github.com/TotalFreedom/TotalFreedomMod/issues/266

### CODE REVIEW

File: src/me/StevenLawson/TotalFreedomMod/Listener/TFM_PlayerListener.java [hunk 1]
```diff
@@ -923,4 +928,21 @@ public void onPlayerLogin(PlayerLoginEvent event)
...
+    
+    @EventHandler
+    public void onPlayerConsumePotion(PlayerItemConsumeEvent event)
+    {
+        if (event.getItem().getType() == Material.POTION)
+        {
+            Collection<PotionEffect> fx = Potion.fromItemStack(event.getItem()).getEffects();
+            for (PotionEffect effect : fx)
+            {
+                if (effect.getType() == PotionEffectType.INVISIBILITY && !TFM_AdminList.isSuperAdmin(event.getPlayer()))
```
Comment (mgejay) [inline src/me/StevenLawson/TotalFreedomMod/Listener/TFM_PlayerListener.java]: It is better to avoid nested if statements.

**Comment by AvalancheYT** [comment]
Invisible is allowed.

**Comment by LegendIsAwesomes** [comment]
@AvalancheYT the issue is accepted, so your wrong.

**Comment by LegendIsAwesomes** [comment]
@MrImmaculate There. I fixed the nested if statements.

**Comment by AvalancheYT** [comment]
@LegendIsAwesomes, according to our server rules & conduct policy, invisible is allowed.
```e. Players can use invisibility although admins may sometimes remove it in mass (inc. /invis smite) for security purposes.```

Read more: http://totalfreedom.boards.net/thread/21065/server-rules-conduct-policy-everybody#ixzz3zzlpfH1M

**Comment by LegendIsAwesomes** [comment]
I don't care; I'm doing what the issue says.

**Comment by Wild1145** [comment]
I think you might have misunderstood the aim of the issue. The aim was to allow only admins to access game mode 3, not to block inis...

**Comment by MrImmaculate** [comment]
@Wild1145 is correct. Please append this PR according to the issue.

**Comment by LegendIsAwesomes** [comment]
Ahh okay, Will fix.

**Comment by LegendIsAwesomes** [comment]
@JeromSar was @Wild1145 right?

**Comment by JeromSar** [comment]
@LegendIsAwesomes He is partially right. He is referring to another issue. This PR should address /invis, a command for admins

**Comment by LegendIsAwesomes** [comment]
@JeromSar I agreed to the CLA.

**Comment by JeromSar** [comment]
@LegendIsAwesomes Great!

**Review by Wild1145 — COMMENTED** [review]
This PR now requires testing. At a glance it looks good.

**Review by Wild1145 — CHANGES_REQUESTED** [review]
Merge conflicts.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,97 +1 @@
...
-* Please refrain from using an excessive amount of commits. As few as possible is generally the best.
-* Please do not spread your contribution over several pull-requests.
-
-## Additional Resources ##
-* [TotalFreedom information](http://totalfreedom.me)
-* [TotalFreedom forums](http://totalfreedom.boards.net)
-* [Bug tracker](https://github.com/TotalFreedom/TotalFreedomMod/issues)
-* [General GitHub documentation](http://help.github.com/)
-* [GitHub pull request documentation](http://help.github.com/send-pull-requests/)
+Visit [the wiki page](https://github.com/TotalFreedom/TotalFreedomMod/wiki/Contributing) to see the contributing guidelines.
```
Comment (Wild1145) [inline CONTRIBUTING.md:1]: Please revert this change.

**Review by Wild1145 — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
@Wild1145 This can be closed as well, it doesn't resolve the issue given and is made for TFM 43

**Comment by Wild1145** [comment]
This one I will leave open until another PR with the same functionality has been merged, as this could still be rebased to work.

**Comment by ghost** [comment]
But @Wild1145 , this doesn't help the server at all, it literally blocks the invis potion

**Comment by Wild1145** [comment]
Okay, I've had a look through the associated issues and the ones that then have superseded it. Currently #856 is the issue this should have resolved. The scope of the original issue has been changed, and this PR would no longer resolve the associated issue. Given also that rules have been changed around invis, I would agree and will close this PR.

**Pull request closed without merging**